src/client/core/utils/dom.js

```diff
@@ -390,8 +390,16 @@ export function isTopWindow (win) {
     }
 }
 
-export function findIframeByWindow (iframeWindow, iframeDestinationWindow) {
-    const iframes = (iframeDestinationWindow || window).document.getElementsByTagName('iframe');
+export function findIframeByWindow (iframeWindow) {
+    const iframes = [];
+
+    document.querySelectorAll('*').forEach(elem => {
+        if (elem.tagName === 'IFRAME')
+            iframes.push(elem);
+
+        if (elem.shadowRoot)
```

Comment:
Avoid using the the `forEach` method here. It can be overridden and it's not cross-browser code (we are still supporting IE).